development_helpers/Dockerfile.dev

```diff
@@ -1,56 +1,56 @@
 
 WORKDIR /opt/CTFd
+COPY --chown=1001:1001 . .
 
 # hadolint ignore=DL3008
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends \
-        build-essential \
-        libffi-dev \
-        libssl-dev \
-        git \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/* \
-    && python -m venv /opt/venv
+
+# TODO Won't run as non root user
+# RUN apt-get update \
+#     && apt-get install -y --no-install-recommends \
+#         build-essential \
+#         libffi-dev \
+#         libssl-dev \
+#         git \
+#     && apt-get clean \
+#     && rm -rf /var/lib/apt/lists/* \
+#     && python -m venv /opt/venv
 
 ENV PATH="/opt/venv/bin:$PATH"
 
-COPY . /opt/CTFd
+COPY ../ /opt/CTFd/plugins/llm_verification
+
+# RUN pip install --no-cache-dir -r requirements.txt \
+#     && for d in CTFd/plugins/*; do \
+#         if [ -f "$d/requirements.txt" ]; then \
+#             pip install --no-cache-dir -r "$d/requirements.txt";\
+#         fi; \
+#     done;
+
+# User already exists
+# RUN useradd \
+#     --no-log-init \
+#     --shell /bin/bash \
+#     -u 1001 \
+#     ctfd \
+#     && mkdir -p /var/log/CTFd /var/uploads \
+#     && chown -R 1001:1001 /var/log/CTFd /var/uploads /opt/CTFd \
+#     && chmod +x /opt/CTFd/docker-entrypoint.sh
 
-RUN pip install --no-cache-dir -r requirements.txt \
-    && for d in CTFd/plugins/*; do \
-        if [ -f "$d/requirements.txt" ]; then \
-            pip install --no-cache-dir -r "$d/requirements.txt";\
-        fi; \
-    done;
+ENV PATH="/opt/venv/bin:$PATH"
 
+USER root
+RUN chown -R 1001:1001 /var/log/CTFd /var/uploads /opt/CTFd 
 
-FROM python:3.9-slim-buster as release
-WORKDIR /opt/CTFd
-
-# hadolint ignore=DL3008
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends \
-        libffi6 \
-        libssl1.1 \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
-
-COPY --chown=1001:1001 . /opt/CTFd
-
-RUN useradd \
-    --no-log-init \
-    --shell /bin/bash \
-    -u 1001 \
-    ctfd \
-    && mkdir -p /var/log/CTFd /var/uploads \
-    && chown -R 1001:1001 /var/log/CTFd /var/uploads /opt/CTFd \
-    && chmod +x /opt/CTFd/docker-entrypoint.sh
-
-COPY --chown=1001:1001 --from=build /opt/venv /opt/venv
-ENV PATH="/opt/venv/bin:$PATH"
+# The directory from which the compose file is called on the host OS matters
+# Currently running docker compose -f development_helpers/docker-compose.dev.yml build 
+RUN chmod +x /opt/CTFd/plugins/llm_verification/development_helpers/docker-entrypoint.dev.sh
+RUN chown 1001:1001 /opt/CTFd/plugins/llm_verification/development_helpers/docker-entrypoint.dev.sh
 
 USER 1001
 EXPOSE 8000
+
 # LLMV: Modified entrypoint to use development entrypoint.
-ENTRYPOINT ["/opt/CTFd/CTFd/plugins/llm_verification/development_helpers/docker-entrypoint.dev.sh"]
+RUN ls -la /opt/CTFd/plugins/llm_verification/development_helpers
+ENTRYPOINT ["/opt/CTFd/plugins/llm_verification/development_helpers/docker-entrypoint.dev.sh"]
```

Comment:
Still getting an entry point error. Im baffled as to what the issue is

```
docker compose -f development_helpers/docker-compose.dev.yml up
docker compose -f development_helpers/docker-compose.dev.yml up
```

<img width="1680" alt="image" src="https://github.com/aivillage/llm_verification/assets/7213793/da172887-7b5a-4015-88c8-25da88be3673">